doc/user-guides/secure-protect-connect-single-multi-cluster.md

```diff
@@ -377,34 +387,34 @@ EOF
 Check your Gateway policies are enforced as follows:
 
 ```bash
-kubectl get dnspolicy ${gatewayName}-dnspolicy -n ${gatewayNS} -o=jsonpath='{.status.conditions[?(@.type=="Enforced")].message}'
-kubectl get authpolicy ${gatewayName}-auth -n ${gatewayNS} -o=jsonpath='{.status.conditions[?(@.type=="Enforced")].message}'
-kubectl get ratelimitpolicy ${gatewayName}-rlp -n ${gatewayNS} -o=jsonpath='{.status.conditions[?(@.type=="Enforced")].message}'
+kubectl get dnspolicy ${GATEWAY_NAME}-dnspolicy -n ${GATEWAY_NS} -o=jsonpath='{.status.conditions[?(@.type=="Enforced")].message}'
+kubectl get authpolicy ${GATEWAY_NAME}-auth -n ${GATEWAY_NS} -o=jsonpath='{.status.conditions[?(@.type=="Enforced")].message}'
+kubectl get ratelimitpolicy ${GATEWAY_NAME}-rlp -n ${GATEWAY_NS} -o=jsonpath='{.status.conditions[?(@.type=="Enforced")].message}'
 ```
 
 Check your listener is ready as follows:
 
 ```
-kubectl get gateway ${gatewayName} -n ${gatewayNS} -o=jsonpath='{.status.listeners[0].conditions[?(@.type=="Programmed")].message}'
+kubectl get gateway ${GATEWAY_NAME} -n ${GATEWAY_NS} -o=jsonpath='{.status.listeners[0].conditions[?(@.type=="Programmed")].message}'
 ```
 
 ### Step 6 - Test connectivity and deny all auth 
 
 You can use `curl` to hit your endpoint. You should see a `403`. Because this example uses Let's Encrypt staging, you can pass the `-k` flag:
 
 ```bash
-curl -k -w "%{http_code}" https://$(kubectl get httproute test -n ${gatewayNS} -o=jsonpath='{.spec.hostnames[0]}')
+curl -k -w "%{http_code}" "https://$(kubectl get httproute test -n ${GATEWAY_NS} -o=jsonpath='{.spec.hostnames[0]}')"
 ```
 
 ### Step 7 - Opening up the Gateway for other namespaces
 
 Because you have configured the Gateway, secured it with Kuadrant policies, and tested it, you can now open it up for use by other teams in other namespaces:
 
 ```bash
-kubectl patch gateway ${gatewayName} -n ${gatewayNS} --type='json' -p='[{"op": "replace", "path": "/spec/listeners/0/allowedRoutes/namespaces/from", "value":"All"}]'
+kubectl patch gateway ${GATEWAY_NAME} -n ${GATEWAY_NS} --type='json' -p='[{"op": "replace", "path": "/spec/listeners/0/allowedRoutes/namespaces/from", "value":"All"}]'
 ```
 
-### Step 8 - Extending this Gateway to multiple clusters and configuring geo-based routing
+### Step 8 - Extending this Gateway to multiple clusters and configuring geo-based routing (Optional)
```

Comment:
This is no longer true. You just create a new Gateway and new DNSPolicy https://github.com/Kuadrant/kuadrant-operator/blob/main/doc/user-guides/dnspolicy/load-balanced-dns.md

Reply:
This whole section can prob be removed to be honest for now.